setup.rst

```diff
@@ -37,32 +37,36 @@ In other words, your new app is ready!
 Running your Symfony Application
 --------------------------------
 
-On production, you should use a web server like Nginx or Apache
-(see :doc:`configuring a web server to run Symfony </setup/web_server_configuration>`).
-But for development, it's convenient to use the :doc:`Symfony Local Web Server <setup/symfony_server>`.
+On production, you should use a web server like Nginx or Apache (see
+:doc:`configuring a web server to run Symfony </setup/web_server_configuration>`).
+But for development, it's better to use the :doc:`Symfony Local Web Server <setup/symfony_server>`.
```

Comment:
I think this change is questionable, "better" is definitely an opinion which is debatable, but "convenient" sounds reasonable.